proposals/dao/vip_9.ts

```diff
@@ -0,0 +1,90 @@
+import { ethers } from 'hardhat';
+import { expect } from 'chai';
+import {
+  DeployUpgradeFunc,
+  NamedAddresses,
+  SetupUpgradeFunc,
+  TeardownUpgradeFunc,
+  ValidateUpgradeFunc
+} from '@custom-types/types';
+import { getImpersonatedSigner } from '@test/helpers';
+
+/*
+
+Timelock Proposal #9
+
+Description: Add DAI, FEI and USDC Compound PCV Deposits to the PCV Guardian
+
+Steps:
+  1 - Deploy all Compound PCV Deposits
+  2 - Add them to the PCV Guardian whitelist
+
+*/
+
+const vipNumber = '9';
+
+// Do any deployments
+// This should exclusively include new contract deployments
+const deploy: DeployUpgradeFunc = async (deployAddress: string, addresses: NamedAddresses, logging: boolean) => {
+  const erc20CompoundPCVDepositFactory = await ethers.getContractFactory('ERC20CompoundPCVDeposit');
+
+  const daiCompoundPCVDeposit = await erc20CompoundPCVDepositFactory.deploy(addresses.core, addresses.cDai);
+  const feiCompoundPCVDeposit = await erc20CompoundPCVDepositFactory.deploy(addresses.core, addresses.cFei);
+  const usdcCompoundPCVDeposit = await erc20CompoundPCVDepositFactory.deploy(addresses.core, addresses.cUsdc);
+
+  await daiCompoundPCVDeposit.deployed();
+  await feiCompoundPCVDeposit.deployed();
+  await usdcCompoundPCVDeposit.deployed();
```

Comment:
wait on each deployment before going to the next one to prevent nonce mismatches and script errors on mainnet